docs/README.md

```diff
@@ -1,6 +1,40 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the auth-service's schema:
+
+<p align="center">
+  <img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/spikerheado1234/auth-service/master/docs/ErDiagram.puml">
+</p>
+
+The following table depicts the types of each attribute in the `Consent` table along with the constraints:
+
+|Attribute| Represented Internal Type | Constraints| Description |
+|:---------:|:---------------------------:|:------------:|:---------------------------:|
+|`id`|`VARCHAR(36)`|Unique - Primary Key|Alphanumeric consent ID|
+|`initiator_id`|`VARCHAR(32)`|Non-Null|Alphanumeric PISP ID|
+|`participant_id`|`VARCHAR(32)`|Non-Null|Alphanumeric DFSP ID|
+|`credential_id`|`INT UNSIGNED`|Unique - Non-Null|Unsigned numeric ID - upto 2^32 records|
+|`credential_type`|`VARCHAR(16)`|Non-Null|Alphanumeric enum value|
+|`credential_status`|`VARCHAR(10)`|Non-Null|String - `PENDING`/`ACTIVE`|
+|`credential_payload`|`TEXT`|Can be Null|Public key string|
+|`credential_challenge`|`CHAR(128)`|Non-Null|Base 64 encoded challenge string|
+
+The following table depicts the types of each attribute in the `Scope` table:
+
+|Attribute| Represented Internal Type | Constraints| Description |
+|:---------:|:---------------------------:|:------------:|:---------------------------:|
+|`id`|`VARCHAR(36)`|Unique - Primary Key|Alphanumeric Scope ID|
+|`consent_id`|`VARCHAR(36)`|Non-Null - Foreign Key|Foreign Key for `id` in the `Consent` table|
+|`action`|`VARCHAR(36)`|Non-Null|Account scope (enum) action allowed with corresponding consent|
+|`account_number`|`VARCHAR(36)`|Non-Null|Account number for associated scope|
```

Comment:
I think this should be `account_id`, and not account number. The `account_id` is an opaque identifier provided by the DFSP, and calling it 'number' is a little too opinionated, in my opinion ;)